fdbserver/include/fdbserver/TagThrottler.h

```diff
@@ -100,4 +100,6 @@ class GlobalTagThrottler : public ITagThrottler {
 public:
 	void setQuota(TransactionTagRef, ThrottleApi::TagQuotaValue const&);
 	void removeQuota(TransactionTagRef);
+	void removeExpiredTags();
```

Comment:
Are these functions being exposed just for unit testing? If so, would it make sense to add a comment to that effect or name the functions TESTING_* or equivalent? (I don't know what the FDB convention is.)

Reply:
There is a comment above in line 99 that these are public only for testing. I didn't rename the method to include "testing" because it is used internally for non-testing purposes (i.e. `removeExpiredTags` would be private if not for the unit tests).